src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -206,10 +206,12 @@ string MakeHelixWorkItemProject(WorkItemInfo workItemInfo)
 
                 AddRehydrateTestFoldersCommand(command, workItemInfo, isUnix);
 
+                var xmlResultsFilePath = ProcessTestExecutor.GetResultsFilePath(workItemInfo, options, "xml");
+
                 // Build an rsp file to send to dotnet test that contains all the assemblies and tests to run.
                 // This gets around command line length limitations and avoids weird escaping issues.
                 // See https://docs.microsoft.com/en-us/dotnet/standard/commandline/syntax#response-files
-                var rspFileContents = ProcessTestExecutor.BuildRspFileContents(workItemInfo, options);
+                var rspFileContents = ProcessTestExecutor.BuildRspFileContents(workItemInfo, options, xmlResultsFilePath, htmlResultsFilePath: null);
```

Comment:
It looks like this doesn't respect the `options.IncludeHtml` setting. Is that fine? Should we assert that the option is false on this code path?

Reply:
this option is never set for helix.  Will assert that it is false